tools/myst/host/exec.c

```diff
@@ -204,31 +185,24 @@ int exec_launch_enclave(
     return retval;
 }
 
-#define USAGE_EXEC_SGX \
-    "\
+#define USAGE_EXEC_SGX "\n\
 \n\
-Usage: %s exec-sgx <rootfs> [options] <application> <app_args...>\n\
+Usage: %s exec-linux [options] <rootfs> <application> <args...>\n\
```

Comment:
This usage string is for SGX target, isn't it?

Reply:
Fixed, thanks.